src/parser/jobs/rdm/Gauge.js

```diff
@@ -180,8 +203,8 @@ export default class Gauge extends Module {
 
 		_onCast(event) {
 			const abilityId = event.ability.guid
-			//This just lets us determine if we've modified the current Mana numbers at all
-			//const isCalculated = false
+			//Handle Finisher BEFORE adjusting mana
+			this._handleFinisher(abilityId)
```

Comment:
This seems like a lot of stuff to be put in one module - finisher suggestions feel new-module-esque.

Not sure on the semantics of ensuring the finisher handler gets processed _before_ calculating gauge changes, ping me about it.